@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
+    <div class="host-network-table-container">
+      <UiTitle type="h4" class="header">
+        <slot>{{ $t('pifs') }}</slot>
```

Comment:
`slot` is unnecessary here:
```suggestion
        {{ $t('pifs') }}
```